package.json

```diff
@@ -16,12 +16,23 @@
     "format": "prettier --loglevel warn --write '**/*.ts' && tslint --fix --format stylish '**/*.ts' --exclude 'node_modules/**'",
     "test": "npm run lint && npm run unit-test",
     "unit-test": "tap test/system test/lib -R=spec --timeout=300",
-    "prepare": "npm run build"
+    "prepare": "npm run build",
+    "perf": "yarn run ts-node test/perf.ts"
   },
   "dependencies": {
+    "@types/md5": "^2.1.33",
+    "@types/minimatch": "^3.0.3",
+    "@types/tar-fs": "^1.16.1",
+    "@types/tar-stream": "^1.6.0",
+    "@types/tmp": "^0.1.0",
     "debug": "^4.1.1",
     "dockerfile-ast": "0.0.16",
+    "md5": "^2.2.1",
+    "minimatch": "^3.0.4",
     "semver": "^6.1.0",
+    "tar-fs": "^2.0.0",
```

Comment:
I don't think we need both `tar-fs` and `tar-stream`? Same with the `@types`

Reply:
Moved from `dependencies` section to `devDependencies` section since it is used by `test/lib/image-extractor.test.ts`